src/webhooks/mod.rs

```diff
@@ -1,27 +1,47 @@
 use postgres;
 use reqwest;
+use rand::{thread_rng, Rng, distributions::Alphanumeric};
+use sha2::Sha256;
+use hmac::{Hmac, Mac};
+use url::Url;
+use base64;
+
+// Create alias for HMAC-SHA256
+type HmacSha256 = Hmac<Sha256>;
+
 use crate::{
     worker,
     err::HecateError
 };
 
-#[derive(Serialize, Deserialize, Clone)]
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
```

Comment:
Should we remove Debug before deploying?

Reply:
Per chat, going to leave `Debug` as it makes the `Webhooks` struct inspectable. Noting that we haven't been consistent about adding/not adding `Debug` traits to our structures, though we should revisit this in the future.